integreat_cms/static/src/js/analytics/hix-widget.ts

```diff
@@ -64,9 +64,10 @@ const setHixLabelState = (state: string) => {
     }
```

Comment:
Not sure whether that's a bad idea, but what do you think about hiding the button when there is nothing to by adding the block
```
    // Hide the button if there is no content or the value is up-to-date
    const updateButton = document.getElementById("btn-update-hix-value");
    if (state === "updated" || state === "no-content") {
        updateButton.classList.add("hidden");
    } else {
        updateButton.classList.remove("hidden");
    }
```
to the `setHixLabelState()` function? (And remove the `updateButton.disabled = ...` statements?)

Reply:
Sure, why not! 😄